frontend/src/app/modules/boards/board/board-partitioned-page/board-list-container.component.ts

```diff
@@ -21,6 +20,7 @@ import {AddListModalComponent} from "core-app/modules/boards/board/add-list-moda
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {BoardListCrossSelectionService} from "core-app/modules/boards/board/board-list/board-list-cross-selection.service";
 import {filter} from "rxjs/operators";
+import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/e71869ec897476f98b23dceb4c3c60722bb3632a/frontend/src/app/modules/boards/board/board-partitioned-page/board-list-container.component.ts#L24"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>